### PR TITLE
Add SYS_RESOURCE is needed for custom PSPs

### LINKED ISSUE #303: Document that (for custom PSPs) they should specify `SYS_RESOURCE` in added capabilities
https://trello.com/c/5ejfb8Ok/907-document-that-for-custom-psps-they-should-specify-sysresource-in-added-capabilities

### PR DESCRIPTION
**Pull request by btat**
Resolves #303

### CODE REVIEW

File: xml/cap_depl_install_production.xml [hunk 1]
```diff
@@ -594,7 +617,23 @@ secrets:
...
+  diego_brain:
+    capabilities: ["SYS_RESOURCE"]
+  diego_ssh:
+    capabilities: ["SYS_RESOURCE"]
+  nats:
+    capabilities: ["SYS_RESOURCE"]
+  router:
+    capabilities: ["SYS_RESOURCE"]
+  routing_api:
+    capabilities: ["SYS_RESOURCE"]</screen>
```
Comment (f0rmiga) [inline xml/cap_depl_install_production.xml]: It's easier to track the closing XML/HTML tag when it is put on its own line.

File: xml/cap_depl_install_production.xml [hunk 2]
```diff
@@ -434,6 +435,28 @@ clusterrolebinding.rbac.authorization.k8s.io "cap:clusterrole" created</screen>
         <literal>suse.cap.psp</literal>.
     </para>
 </tip>
+   <important>
+    <title>Using Custom PodSecurityPolicies</title>
+    <para>
+     When using a custom PSP, your <filename>scf-config-values.yaml</filename> file will require the <literal>SYS_RESOURCE</literal> capability to be added to some roles as follows:
+    </para>
+<screen>sizing:
```
Comment (f0rmiga) [inline xml/cap_depl_install_production.xml]: `sizing:` can go to the next line and `<screen>` can be indented to match the `</para>`, indicating it is a child of `<important />`.

File: xml/cap_depl_install_production.xml [hunk 3]
```diff
@@ -434,6 +435,28 @@ clusterrolebinding.rbac.authorization.k8s.io "cap:clusterrole" created</screen>
...
+    capabilities: ["SYS_RESOURCE"]
+  diego_ssh:
+    capabilities: ["SYS_RESOURCE"]
+  nats:
+    capabilities: ["SYS_RESOURCE"]
+  router:
+    capabilities: ["SYS_RESOURCE"]
+  routing_api:
+    capabilities: ["SYS_RESOURCE"]
+</screen>
```
Comment (f0rmiga) [inline xml/cap_depl_install_production.xml]: Same here regarding the indentation.

**Comment by btat** [comment]
Thanks @f0rmiga! Changes made for better tag alignment. The line breaks near the `screen` tag is a bit weird though, according to Sven in https://github.com/SUSE/doc-cap/pull/148:

```
line breaks after start screen tags and before end
screen tags (according to our style gurus this is a good thing
for epub).
```

**Pull request merged**